requirements.txt

```diff
@@ -29,24 +29,24 @@ oauth2client>=4.1.3
 ## Necessary since API Client Libraries are not thread-safe
 httplib2shim>=0.0.3
 
-# Azure Provider
-## core requirements
-azure-cli-core==2.12.0
-## for RBAC in AAD
-azure-graphrbac==0.61.1
-adal==1.2.4
-PyJWT==1.7.1  # FIXME - this is currently required due to the MSAL dependency but should be removed once we switch to ADAL
+#for authentication
+azure-identity==1.5.0
+
 ## for resources
-azure-mgmt-storage==7.1.0
-azure-mgmt-monitor==0.5.2
-azure-mgmt-sql==0.11.0
-azure-mgmt-security==0.4.1
-azure-mgmt-keyvault==1.1.0
-azure-mgmt-network==2.5.1
-azure-mgmt-redis==6.0.0
-azure-mgmt-web==0.47.0
-azure-mgmt-compute==12.0.0
-azure-mgmt-authorization==0.60.0
+azure-mgmt-resource==15.0.0
+azure-mgmt-storage==16.0.0
+azure-mgmt-monitor==2.0.0
+azure-mgmt-sql==1.0.0
+azure-mgmt-security==1.0.0
+azure-mgmt-keyvault==8.0.0
+azure-mgmt-network==17.1.0
+azure-mgmt-redis==12.0.0
+azure-mgmt-web==1.0.0
+azure-mgmt-compute==18.2.0
+azure-mgmt-authorization==1.0.0
+
+--extra-index-url https://test.pypi.org/simple
+msgraphcore==0.0.2
```

Comment:
I'm not very happy we have to do this, but on the other hand the alternative is to make HTTP requests ourselves, which I'm not sure is any better.

Reply:
Agreed, as soon as the library is available on the official Pypi, the extra index url should be removed and the library updated

Reply:
Let's mark this as resolved and include a note in the CIS PR to wait until the issue is resolved before merging to `develop`.